README.md

```diff
@@ -99,17 +101,17 @@ An implementation is not compliant if it fails to satisfy one or more of the MUS
 
   <dt>Service Binding</dt>
-  <dd>The act of or representation of the action of providing information about a Service to a workload</dd>
+  <dd>The act of or representation of the action of providing information about a Service to a Workload</dd>
 
   <dt>Secret</dt>
   <dd>A Kubernetes <a href="https://kubernetes.io/docs/concepts/configuration/secret/"><code>Secret</code></a></dd>
 </dl>
 
 # Provisioned Service
 
-A Provisioned Service resource **MUST** define a `.status.binding` which is a `LocalObjectReference`-able (containing a single field `name`) to a `Secret`.  The `Secret` **MUST** exist in the same namespace as the resource.  The `Secret` data **SHOULD** contain a `type` entry with a value that identifies the abstract classification of the binding.  The `Secret` type (`.type` verses `.data.type`) **SHOULD** reflect this value as `servicebinding.io/{type}`, replacing `{type}` with the `Secret` data type.  It is **RECOMMENDED** that the `Secret` data also contain a `provider` entry with a value that identifies the provider of the binding.  The `Secret` data **MAY** contain any other entry.  To facilitate discoverability, it is **RECOMMENDED** that a `CustomResourceDefinition` exposing a Provisioned Service add `servicebinding.io/provisioned-service: "true"` as a label.
+A Provisioned Service resource **MUST** define a `.status.binding` field which is a `LocalObjectReference`-able (containing a single field `name`) to a `Secret`.  The `Secret` **MUST** exist in the same namespace as the resource.  The `Secret` data **SHOULD** contain a `type` entry with a value that identifies the abstract classification of the binding.  The `Secret` type (`.type` verses `.data.type` fields) **SHOULD** reflect this value as `servicebinding.io/{type}`, substituting `{type}` with the `Secret` data type.  It is **RECOMMENDED** that the `Secret` data also contain a `provider` entry with a value that identifies the provider of the binding.  The `Secret` data **MAY** contain any other entry.  To facilitate discoverability, it is **RECOMMENDED** that a `CustomResourceDefinition` exposing a Provisioned Service add `servicebinding.io/provisioned-service: "true"` as a label.
 
-> Note: While the Provisioned Service referenced `Secret` data should contain a `type` entry, the `type` must be defined before it is projected into a workload. This allows a mapping to enrich an existing secret.
+> Note: While the Provisioned Service referenced `Secret` data should contain a `type` entry, the `type` must be defined as it is projected to a workload. The relaxation of the requirement for provisioned services allows for a mapping to enrich an existing secret.
```

Comment:
I hope mentioning a common scenario would be okay.
```suggestion
> Note: While the Provisioned Service referenced `Secret` data should contain a `type` entry, the `type` must be defined as it is projected to a workload. The relaxation of the requirement for provisioned services allows a mapping to enrich an existing secret. For example, ServiceBinding has fields to override `type` and `provider` values.
```

Reply:
```suggestion
> Note: While the Provisioned Service referenced `Secret` data should contain a `type` entry, the `type` must be defined as it is projected to a workload. The relaxation of the requirement for provisioned services allows for a mapping to enrich an existing secret.  For example, `ServiceBinding` has fields to override `type` and `provider` values.
```